packages/financial-templates-lib/src/helpers/math.js

```diff
@@ -0,0 +1,37 @@
+const assert = require("assert");
```

Comment:
🤩 

Will be more to add to this in the future for sure. thanks for the initial refactor!